SK.API/Controllers/TestValuesController.cs

```diff
@@ -1,59 +1,47 @@
-﻿using MediatR;
-using Microsoft.AspNetCore.Mvc;
-using SK.Application.TestValues.Commands.Create;
-using SK.Application.TestValues.Commands.Delete;
-using SK.Application.TestValues.Commands.Edit;
+﻿using Microsoft.AspNetCore.Mvc;
+using SK.Application.TestValues.Commands.CreateTestValue;
+using SK.Application.TestValues.Commands.DeleteTestValue;
+using SK.Application.TestValues.Commands.EditTestValue;
 using SK.Application.TestValues.Queries;
-using SK.Application.TestValues.Queries.Details;
-using SK.Application.TestValues.Queries.List;
+using SK.Application.TestValues.Queries.DetailsTestValue;
+using SK.Application.TestValues.Queries.ListTestValue;
 using System.Collections.Generic;
 using System.Threading.Tasks;
 
 namespace SK.API.Controllers
 {
     public class TestValuesController : ApiController
     {
-        private readonly IMediator _mediator;
-        public TestValuesController(IMediator mediator)
-        {
-            _mediator = mediator;
-        }
-
         [HttpGet]
         public async Task<ActionResult<List<TestValueDto>>> List()
         {
-            return await _mediator.Send(new ListTestValueQuery.Query());
+            return await Mediator.Send(new ListTestValueQuery());
         }
 
         [HttpGet("{id}")]
         public async Task<ActionResult<TestValueDto>> Details(int id)
         {
-            return await _mediator.Send(new DetailsTestValueQuery.Query { Id = id });
+            return await Mediator.Send(new DetailsTestValueQuery { Id = id });
         }
 
         [HttpPost]
-        public async Task<ActionResult<int>> Create([FromBody] CreateTestValueCommand.Command command)
+        public async Task<ActionResult<int>> Create([FromBody] CreateTestValueCommand command)
         {
             return await Mediator.Send(command);
         }
 
         [HttpPut("{id}")]
-        public async Task<ActionResult> Update(int id, EditTestValueCommand.Command command)
+        public async Task<ActionResult> Update([FromBody] TestValueDto testValue)
         {
-            if (id != command.Id)
-            {
-                return BadRequest();
-            }
-
-            await Mediator.Send(command);
+            await Mediator.Send(new EditTestValueCommand { Id = testValue.Id, Name = testValue.Name});
```

Comment:
use constructor: 
EditTestValueCommand(request.Id, request.name);

rename testValue to request since it's request :)

Reply:
Done.